modules/ROOT/pages/salesforce/salesforce-composite-connector.adoc

```diff
@@ -291,6 +291,197 @@ An example of HTTP input for this flow is:
+
+. In the Mule Palette view, search for *Transform Message*.
+. Drag the *Transform Message* component onto the canvas, to the right of *Listener*.
+. Enter the name `MyNewAccount`:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+%output application/json
+---
+{
+   Name: "MyNewAccount"
+}
+----
+
+=== Add the Pre Create Operation
+
+The `Pre create` operation provides metatdata for creating an object called `NewAccount`. It generates a sub-request for the `executeCompositeBatch` operation:
+
+. In the Mule Palette view, search for *Salesforce* and select the *Salesforce Composite Pre create* operation.
+. Drag the *Pre create* operation to the right of *Transform Message*.
+. Click the green plus icon to the right of the *Connector configuration* field to access the Salesforce Composite global element configuration fields.
+. Complete the fields using variables, as follows:
++
+.Complete fields in the General section to create a reusable global element for Salesforce Composite Connector.
+image::salesforce/salesforce-composite-global-config.png[Salesforce Composite Connector global element configuration]
++
+. In the Pre create properties window, enter `Account` in the *Type* field.
+. In the YAML file `src/main/resources/properties/project_name_dev.yaml`, declare and enter values for the variables in the global element .
+. Load the YAML file as a global element.
+
+=== Add the Second Transform Message Component
+
+This `Transform Message` component sets values for the `ID` and `Type` fields, and then passes these values to the `Pre query` operation:
+
+. Drag a second *Transform Message* component to the right of *Pre create*.
+. Enter the following name-value pairs:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+%output application/json
+---
+{
+"Id": payload.Id,
+"Fields" : ["Id", "Name"],
+"Type": "Account"
+}
+----
+
+=== Add the Pre Query Operation
+
+The `Pre query` operation provides metadata for an object to be queried. It generates a sub-request for the `Execute composite batch` operation:
+
+. Drag a Salesforce Composite *Pre query* operation to the right of the second *Transform Message*.
+. In the *Query* field, enter the following query:
++
+`Select Name from Account WHERE Name LIKE '%:name %'`
+
+=== Add the Third Transform Message Component
+
+This `Transform Message` operation passes the output of the `Pre query` operation to the `Execute composite batch` operation.
+
+. Drag a third *Transform Message* component to the right of *Pre query*.
+. Set the output to `payload`:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+%output application/java
+---
+[
+payload
+]
+----
+
+=== Add the Execute Composite Batch Operation
+
+The  `Execute composite batch` operation executes the sub-requests that the `Pre create` and `Pre query` operations created.
+
+Drag a Salesforce Composite *Execute composite batch* operation to the right of the third *Transform Message*.
+
+=== Add the Fourth Transform Message Component
```

Comment:
Were each of these Level-3 subtopics implied by the text in their container Level-2 topic?
I know they are not listed individually, which is not necessary, but I just want to verify with you that the summary of what the example shows is enough to prepare the audience for all these subtasks.